os-eap-probes/added/probes/probe/dmr.py

```diff
@@ -49,6 +49,8 @@ def __readConfig(self):
         self.port = 9990 + int(os.getenv('PORT_OFFSET', 0))
         self.user = os.getenv('ADMIN_USERNAME')
         self.password = os.getenv('ADMIN_PASSWORD')
+        if self.password != "" and self.user == "":
```

Comment:
If you set just ADMIN_PASSWORD=whatever in the template then you get as ENVs in the pod:
 
sh-4.2$ env | grep ADMIN
ADMIN_USERNAME=
ADMIN_PASSWORD=whatever

The probe now detects that condition and sets ADMIN_USERNAME=eapadmin. It detects ADMIN_USERNAME as an empty string, not as null